ios/RNGoogleSignin.m

```diff
@@ -102,6 +102,19 @@ - (dispatch_queue_t)methodQueue
   }];
 }
 
+RCT_EXPORT_METHOD(signInWithAdditionalScopes:(NSDictionary *)options
```

Comment:
there is no need to add another method, let's please use the existing `signIn` method, thank you 🙂

Reply:
done